shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -333,6 +337,9 @@ object GlutenConfig {
   // Batch size.
   val GLUTEN_MAX_BATCH_SIZE_KEY = "spark.gluten.sql.columnar.maxBatchSize"
 
+  // Shuffle Writer buffer size.
+  val GLUTEN_SHUFFLE_WRITER_BUFFER_SIZE = "spark.gluten.shufflewriter.bufferSize"
```

Comment:
"spark.gluten.shufflewriter.bufferSize" --> better to rename to  "spark.gluten.shuffleWriter.bufferSize"

Reply:
I opened PR to fix this typo https://github.com/oap-project/gluten/pull/3073